lib/revisement/analyzer.rb

```diff
@@ -163,14 +162,14 @@ def augmenting_paths_by_target
     #   annotated according to the given changeset
     def annotated_targets
       workspace_modification = find_workspace_modification_in_changesets
-      project_changes = Hash[projects.map do |project|
-        [project, find_project_modification_in_changesets(project: project) || workspace_modification]
-      end]
+      project_changes = projects.to_h do |project|
+        [project, find_project_modification_in_changesets(project:) || workspace_modification]
```

Comment:
TIL about this Ruby 3.1 feature about omitting the matching keyword variable name:
https://bugs.ruby-lang.org/issues/14579

Reply:
Yeah, it's honestly a little unclear when reading, but Rubocop enforces it as a default